### ISSUE TITLE
Suggestion/Bug: For local games, let the program walk through (sub)directories

### ISSUE THREAD

**Issue by lorddusk** [gibaBR/Switch-Backup-Manager#2]
As the title says, select a root folder, and let the program look for .xci files through all sub directories of this folder.

**Comment by lorddusk** [comment]
Nevermind, I'm blind >.<

**Comment by gibaBR** [comment]
But it was made to work with subdirectories. Just tested it againnow  to be sure and it foud my files.
Use Local Files -> Add ->Folder
Didn't work for you?

**Comment by lorddusk** [comment]
Nope, I see him running, but nothing gets added.

**Comment by gibaBR** [comment]
Could you please put here the folder structure you are using so I can try to replicate the problem?

**Comment by KaLoF87** [comment]
I have the same issue as lorddusk.
I noticed that the problem in my case only appear when trying to add directory from my NAS.
When I tried to add a directory with subfolders localy on my hdd, it worked!

**Comment by gibaBR** [comment]
OK! Now I can see the problem.

**Comment by gibaBR** [comment]
KaLoF87, your NAS uses SMB to share files? 
I have a NAS here too and to me it worked like a charm. 
Tested using an assigned letter do the share

and directly pointing to the share itself:
\\mycloud\Games\Switch\XCI\game-a\game-a.xci
\\mycloud\Games\Switch\XCI\game-b\game-b.xci
\\mycloud\Games\Switch\XCI\game-c\folder-x\game-b.xci

Both worked. Can you give me an example of your share?

I really cant replicate the error.

**Comment by KaLoF87** [comment]
Yes im using SMB as well.
I have almost the same file structure:
\\MarcusNAS\Spel\Nintendo\Switch\Game1\Game1,xci
\\MarcusNAS\Spel\Nintendo\Switch\Game2\Game2,xci
\\MarcusNAS\Spel\Nintendo\Switch\Game3\Game3,xci

As i said in the previous post, it worked when I only added 4 games in a folder on my NAS. but not when I added the folder containing almost 100 games.

I'm copying the whole Switch folder now from my NAS to a local disk to rule out if the problem is when adding a lots of games at once.
I will write and update when i tried it.
